```
In file included from src/blob.h:34,
                 from src/blob.c:30:
src/types.h:36:2: error: #error You need a compatible libgit2 version (1.3.x)
   36 | #error You need a compatible libgit2 version (1.3.x)
      |  ^~~~~
error: command 'gcc' failed with exit status 1
```

--- a/extra-python/pygit2/spec
+++ b/extra-python/pygit2/spec
@@ -1,5 +1,4 @@
-VER=1.0.3
+VER=1.7.0
 SRCS="tbl::https://github.com/libgit2/pygit2/archive/v$VER.tar.gz"
-CHKSUMS="sha256::d6428fd4cd5263c76842675519b85874cfbb79db02bc32523eb784573c328a5b"
-REL=2
+CHKSUMS="sha256::8fbd1b91bd594f5f9cdd515e3a691ec148fd431838455487663d73387183d844"
 CHKUPDATE="anitya::id=3985"

